seg_lapa/config/trainer/trainer.yaml

```diff
@@ -0,0 +1,11 @@
+# @package _group_
+name: trainer
+gpus: "0"
+overfit_batches: 0.0
+distributed_backend: "ddp"
+num_nodes: 1
```

Comment:
I believe we're always going to use a single node (don't have multiple machines across which to train). So we won't need to modify this value.
```suggestion
```